### ISSUE TITLE
Loading spine assets from atlas

### ISSUE THREAD

**Issue by lukz** [orange-games/phaser-spine#45]
Hi!

Is there any way to point to other atlas as source of assets for skeleton while loading skeleton?

**Comment by AleBles** [comment]
At the moment, no. It'll only (dynamicly) include the image files defined in the .atlas.

Any suggestion in a different set of loading? I assume you'd prefer to have the spine included in atlas file that has more sprite available than just the spine ones?

**Comment by lukz** [comment]
Yup you're right. It can help reducing load time and with optimization (less texture bindings). Also current way is problematic for webpack as by default it changes filenames to hashes and currently plugin is trying to load png and atlas files of the same filename.

I would like to be able to specify atlas key containing those images (and other) while loading skeleton json file.

**Comment by KidScripty** [comment]
Has this ability been implemented yet?
